app/backend/requirements.txt

```diff
@@ -11,3 +11,5 @@ azure-monitor-opentelemetry==1.0.0b15
 opentelemetry-instrumentation-asgi==0.40b0
 opentelemetry-instrumentation-requests==0.40b0
 opentelemetry-instrumentation-aiohttp-client==0.40b0
+msal==1.23.0
+msgraph-sdk==1.0.0a14
```

Comment:
Their pypi says not to use the SDK in production environments, has that changed? Thats why my PR uses HTTP calls instead. https://pypi.org/project/msgraph-sdk/

Reply:
this is good feedback - i can remove the graph SDK and page through the results

Reply:
sdk removed, replaced with urllib3 usage similar to your PR